c2rust-transpile/src/translator/mod.rs

```diff
@@ -1052,7 +1047,7 @@ fn arrange_header(t: &Translation, is_binary: bool) -> (Vec<syn::Attribute>, Vec
     } else {
         let pragmas = t.get_pragmas();
         for (key, mut values) in pragmas {
-            values.sort();
+            values.sort_unstable();
```

Comment:
How do we know an unstable sort is okay?

Reply:
Clippy told me ) More seriously, the difference between stable and unstable sorting is that unstable sort doesn't need to preserve the order of equal elements. But for primitives, including the `&str` type used here, the elements compare equal under `PartialEq::eq` only if they are identical. For this reason there is literally no difference between stable and unstable sort, but unstable sort is faster.

The [lint description](https://rust-lang.github.io/rust-clippy/master/#stable_sort_primitive) says that sometimes stable sort of primitives is faster than unstable sort, but in the absence of benchmark data I would always go with the lint.

Reply:
I think we'll just leave it as `.sort_unstable()`.  It's unlikely to become a problem I think.